### DISCUSSION TITLE
Dual antigen retrieval protocol (pH 6 and pH 9)

### DISCUSSION THREAD

**Discussion by zmikulski** [IBEXImagingCommunity/ibex_imaging_knowledge_base#57] [Q&A] [answered]
Hi All, 

I'm looking for the dual antigen retrieval protocol that was mentioned in the original DIVE paper. The references pointed me toward a very interesting patent but not an actual protocol. Google search failed to provide anything useful. 

If there is a paper discussing this (or similar approaches) in more detail I'd love to read it. There is a lot of work done for a handful of markers comparing pH6 and ph9 or various retrieval methods, but I struggled to find something with a bit more systematic approach. If you know of something please let me know.

Tagging @radtkea 😊

TY!!

**Comment by radtkea** [comment, marked as answer]
Greetings Zmikulski!

Very happy to answer your question. It's an excellent one! Thank you for asking it in the discussion forum.

A friend and awesome colleague, Liz, published a very detailed protocol for dual antigen retrieval for the Cell DIVE. I haven't followed her protocol, but I trust her expertise! 
https://www.protocols.io/view/cell-dive-platform-slide-clearing-and-antigen-retr-bp2l6b9yrgqe/v1

I am totally converted to the notion of dual antigen retrieval for highly multiplexed imaging experiments. I hope to contribute a protocol to the knowledge-base that compares dual antigen retrieval performed using a water bath (low cost), pressure cooker, and Leica Bond. ETA: Mid-June based on other commitments.

Regarding a systematic approach comparing different retrieval conditions: Check out Nordic immunohistochemical Quality Control (NordiQC). They are geared to clinical pathology, but I think you will appreciate the care and effort behind their great work. Truly a treasure: https://www.nordiqc.org/epitope.php

Happy imaging and best of luck!
A

**Reply by zmikulski** [reply]
Thank you very much! That protocol is a great starting point, but we'll probably simplify a few things. When we got our NxGen decloaker we compared it with a household pressure cooker and the results were quite similar. We upgraded mostly for more advanced reporting options and hoping that it will be more reproducible over many runs. So far so good, but it would be awesome to see a comparison across modern antigen retrieval hardware. We might be getting data comparing microwave-based EZ-Retriever and NxGen. 

The NordiQC is indeed awesome, I have forgotten about them. With some good winds, I hope the community will have some data to put together for mouse FFPE resources as well. 

Best regards, Z.

**Reply by radtkea** [reply]
Hi Z:)

You're very welcome! Happy Liz's protocol could help you!

Wow, that's a great find. A household pressure cooker will be more affordable and accessible to more scientists around the world.

We would really love to see your results with the EZ-Retriever and NxGen. Fingers crossed. I am just planning a simple test of the different methods/hardware available to us. Will keep you posted!

Re: NordiQC. Indeed! My friend, Christopher Werlein, told me about them. Love the rigor!

Mouse FFPE: Oh yes. I have tried to avoid this format for ages. Suggestion: Reach out to the discussion forum/antibody suppliers to ask for more mouse FFPE antibodies? 

Have a lovely day! Thanks for all your enthusiasm/activity in the discussion forum!
A
